test/services/AppealServiceSpec.scala

```diff
@@ -446,11 +446,28 @@ class AppealServiceSpec extends SpecBase with LogCapturing {
       }
     }
 
-    "succeed if one of 2 appeal submissions fail and log a PD (LPP2 fails)" in new Setup {
+    "succeed if an error occurs during file notification storage and log a PD (LPP1 fails)" in new Setup {
+      when(mockPenaltiesConnector.submitAppeal(any(), any(), any(), ArgumentMatchers.eq("123456789"), any(), any())(any(), any()))
+        .thenReturn(Future.successful(HttpResponse(MULTI_STATUS, "Some issue with document storage")))
+      when(mockPenaltiesConnector.submitAppeal(any(), any(), any(), ArgumentMatchers.eq("123456788"), any(), any())(any(), any()))
+        .thenReturn(Future.successful(HttpResponse(OK, "PR-1234")))
```

Comment:
What if one was MULTI_STATUS and the other was ISE, then it would show the error page, see above comment for fix